src/main.c

```diff
@@ -1922,10 +209,29 @@ unsigned char io_event(unsigned char channel) {
 #endif
 
     case SEPROXYHAL_TAG_DISPLAY_PROCESSED_EVENT:
+#ifdef HAVE_BAGL
         UX_DISPLAYED_EVENT({});
+#endif  // HAVE_BAGL
+#ifdef HAVE_NBGL
```

Comment:
NBGL stuff should not be part of this commit?

Reply:
oops